pennylane/labs/resource_estimation/ops/qubit/non_parametric_ops.py

```diff
@@ -35,6 +35,25 @@ def resource_rep(cls) -> re.CompressedResourceOp:
         return re.CompressedResourceOp(cls, {})
 
 
+class ResourceS(qml.S, re.ResourceOperator):
```

Comment:
You are missing implementations for PauliX, PauliY and PauliZ. Note that while their PL decomposition isn't very helpful, they can be converted into combinations of Hadamard, S, and T. See [here](https://en.wikipedia.org/wiki/Clifford_gates) and [here](https://en.wikipedia.org/wiki/Quantum_logic_gate) for more info.